odoo/addons/test_new_api/models/test_new_api.py

```diff
@@ -1406,6 +1406,58 @@ class Group(models.Model):
     user_ids = fields.Many2many('test_new_api.user')
 
 
+class GroupsNoGroupAccess(models.Model):
+    _name = 'test_new_api.groups.no'
```

Comment:
The name is confusing.  This model does not contain groups, it contains records.
```suggestion
class ModelNoAccess(models.Model):
    _name = 'test_new_api.model.no_access'
```